src/server/game/Entities/Player/Player.cpp

```diff
@@ -5018,13 +5023,14 @@ void Player::ApplyRatingMod(CombatRating combatRating, int32 value, bool apply)
 
 void Player::UpdateRating(CombatRating cr)
 {
-    int32 amount = m_baseRatingValue[cr];
+    int32 amount = m_baseRatingValue[cr] + m_bonusRatingValue[cr];
     // Apply bonus from SPELL_AURA_MOD_RATING_FROM_STAT
     // stat used stored in miscValueB for this aura
     AuraEffectList const& modRatingFromStat = GetAuraEffectsByType(SPELL_AURA_MOD_RATING_FROM_STAT);
     for (AuraEffectList::const_iterator i = modRatingFromStat.begin(); i != modRatingFromStat.end(); ++i)
         if ((*i)->GetMiscValue() & (1<<cr))
             amount += int32(CalculatePct(GetStat(Stats((*i)->GetMiscValueB())), (*i)->GetAmount()));
+
```

Comment:
It could have been added for readability, at least before the PR is ready for merging.